src/UI/Seller/src/app/orders/components/line-item-table/line-item-table.component.ts

```diff
@@ -35,10 +35,11 @@ export class LineItemTableComponent {
   _liGroupedByShipFrom: HSLineItem[][]
   _supplierOrders: HSOrder[] = []
   _statusChangeForm = new FormArray([])
-  _tableStatus = LineItemTableStatus.Default
+  _tableStatus: string
```

Comment:
Shouldn't the type be `LineItemTableStatus`?

Reply:
Just updated to use types.